arkane/ess/adapter.py

```diff
@@ -45,6 +45,11 @@ class ESSAdapter(ABC):
     An abstract ESS Adapter class
     """
 
+    def __init__(self, path, check_for_errors=True):
+        self.path = path
+        if check_for_errors:
+            self.check_for_errors()
+
```

Comment:
If possible, squash this commit in to the ones before it, as this fixes a change within the same PR. If that proves difficult though I am fine with leaving this commit the way it is, this is a big ask for little gain.

I think this would mean that this addition to the ess adapter goes into the same commit as adding the check_for_errors method, and the deletions in this commit would be applied to the various other commits in this PR

Reply:
Thanks for catching this. I squashed my many incremental commits so they are now organized by "additions to our ESS log classes" and "additions to our ESS log tests".